### PR TITLE
Fixes merge 17

### PR DESCRIPTION
**Pull request by tonianner**
Fixed all suggestions from previous PR (except for logo resize and <address> (see comments))
Amended the http:// out from the seed.db (to make it more visually appealing on the restaurant profiles)

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -22,7 +22,7 @@
 }
 
 body {
-  background-image: url("background_home.jpeg");
+  background-image: url(/assets/background_home.jpeg);
```
Comment (f3r) [inline app/assets/stylesheets/application.scss]:
``` css
body {
  background: image-path("background_home.jpeg");
}
```

https://github.com/rails/sass-rails#asset-helpers

Comment (f3r) [inline app/assets/stylesheets/application.scss]: When you deploy to heroku, all your images will go through the [Asset Pipeline](http://guides.rubyonrails.org/asset_pipeline.html) and your assets will get [fingerPrinted](http://guides.rubyonrails.org/asset_pipeline.html#what-is-fingerprinting-and-why-should-i-care-questionmark), so the name of the file gets renamed in production, and using `<%= image_tag "logo.jpg" %>` will generate:

``` html
<img 
  src="/assets/logo-82b26e591bbb31280898d6a3bfcac1a305a40c26a3d44fa57c1de374ddc2e093.jpg" 
  alt="Logo 82b26e591bbb31280898d6a3bfcac1a305a40c26a3d44fa57c1de374ddc2e093">
```

**Comment by f3r** [comment]
Let me know when you fix this and I'll merge it :)

**Comment by cryptokoala4** [comment]
Hi Fer, I changed that line to `background-image: image-path("background_home.jpeg");` but now it won't work in localhost. I made 2 lines instead (1 for localhost and 1 for heroku)... 

```
  background-image: url(/assets/background_home.jpeg);
  background-image: image-path("background_home.jpeg");
```

**Comment by cryptokoala4** [comment]
I do not understand why `background-image: image-url("background_home.jpeg");` works for localhost:3000. My background_home.jpeg is a local image stored in my computer, so why will "image-URL" work on localhost when URL means a resource on the internet? (I have no means of testing if anything works on heroku...)

**Comment by f3r** [comment]
@jackywhlee read carefully: https://github.com/wdi-hk-9/Project-SushiMeAway/pull/18#discussion_r43836046

**Comment by jackywhlee** [comment]
I don't get it. I can't style `<%= image_tag "background_home.jpeg"%>` after placing it inside application.html.erb file.

**Comment by cryptokoala4** [comment]
@f3r Fixed it.

**Pull request merged**